src/workerd/io/worker-interface.capnp

```diff
@@ -155,6 +165,8 @@ struct QueueResponse @0x90e98932c0bfc0de {
   # List of Message IDs that were explicitly marked for retry
   explicitAcks @4 :List(Text);
   # List of Message IDs that were explicitly marked as acknowledged
+  retryBatch @5 :QueueRetryBatch;
```

Comment:
Given that this struct is never used for communication outside of just the local process tree, we should be able to rip out the old fields (`retryAll` and `explicitRetries`). Although perhaps even better would be to use the name `explicitRetries` for the `:List(QueueRetryMessage)`, but that's up to you.

Although ripping them out now will require more code changes compared to doing it later (after the JS interfaces can also be cleaned up), so delaying it probably makes sense. But if you don't rip them out now, at least leave a comment in here that they're deprecated and should be removed later.

Reply:
I will remove them in this PR, just need to update queues code first.